routes/index.js

```diff
@@ -21,13 +21,59 @@ router.get("/myhabits", (req, res) => {
     res.redirect("/myhabits.html");
 });
 
-router.use((req, res, next) => {
-    res.status(404).sendFile(join(__dirname, "..", "public", "404.html"));
-});
+// router.use((req, res, next) => {
+//     res.status(404).sendFile(join(__dirname, "..", "public", "404.html"));
+// });
 
 /* ------Katerina end----- */
 
+/* ------Anshul Start ----- */
+router.post("/createHabit", fileUpload(), async (req, res) => {
+    console.log(req.files);
+    if (req.files) {
+        let file;
 
+        file = req.files.img;
+        let uploadPath = "./public/assets/images/" + file.name;
+        console.log(uploadPath);
+        try {
+            // Use the mv() method to place the file somewhere on your server
+            await file.mv(uploadPath, function (err) {
+                if (err) return res.status(500).send(err);
 
+                // res.send("Received!!!");
+            });
+            let { habitName, goalPerDay, startDate, numberOfDays, picture } =
+                req.body;
+            picture = uploadPath;
+            mongo.createHabits(
+                habitName,
+                goalPerDay,
+                startDate,
+                numberOfDays,
+                picture
+            );
+            res.send("Usign custom images");
```

Comment:
after creating new habit, the user is not redirected to myhabits page
```suggestion
           res.redirect("/myhabits.html");
```